content/en/seachat/seachat-manual/02-create-agent/03-advanced-settings/03-agent-memory.md

```diff
@@ -12,13 +12,49 @@ aliases:
    - /en/seachat/seachat-manual/02-create-agent/03-advanced-settings/03-agent-memory
 ---
 
+
+<div style="display: flex; align-self: flex-end; align-items: baseline">
+
+` This feature in only available for: `
+   <div style="border-radius: 30%; 
+    background: linear-gradient(90deg, #135f5c, #3a947b); 
+    width: 5rem; 
+    color: #ffffff; 
+    padding: calc(min(100vw, 1366px)* 0.00439) calc(min(100vw, 1366px)* 0.00878);
+    border-radius: calc(min(100vw, 1366px)* 0.01464);
+    display: block;
+    unicode-bidi: isolate;
+    box-sizing: border-box;
+   font-size: .8rem;
+    align-content: center;
+   ">
+   <div>Enterprise</div>
+</div>
+<div style="border-radius: 30%; 
+    background: linear-gradient(90deg,#824a14,#886f40);
+    width: 5rem; 
+    color: #ffffff; 
+    padding: calc(min(100vw, 1366px)* 0.00439) calc(min(100vw, 1366px)* 0.00878);
+    border-radius: calc(min(100vw, 1366px)* 0.01464);
+    display: block;
+    unicode-bidi: isolate;
+    box-sizing: border-box;
+   font-size: .8rem;
+   margin-left: .5rem;
+        align-content: center;
+   ">
+   <div>Premium</div>
+</div>
+</div>
```

Comment:
Thanks @itIsJim this is really cool. Can you please change the text to:

This feature is only available on <Premium> <Enterprise> plans.